File: 02_Voto.py
idade = input(int("Qual sua idade?\n"))
alfabetizado = input(int("Você é alfabetizado?0-Não, 1-Sim\n"))

if idade < 16:
  print("Não vota")
elif idade > 17 and idade < 71 and alfabetizado == 1:
  print("Voto obrigatorio")
else:
  print("Voto facultativo")
    

File: 02_Voto.c
#include <stdio.h>
int main (){
  int idade;
  int alfabetizado;

  printf("Quantos anos você tem? \n");
  scanf("%d", &idade);
  
  printf("Você é alfabetizado?0-Não, 1-Sim \n");
  scanf("%d", &alfabetizado);

  if (idade < 16){
    printf("Não vota");
  } else if ((idade > 17 && idade < 71) && (alfabetizado == 1)){
    printf("Voto Obrigatorio");
  } else {
    printf("Voto facultativo");
  }
  return 0;
}
